pkg/metrics/otel.go

```diff
@@ -96,7 +96,9 @@ processors:
   filter/cloudmonitoring:
     metrics:
       include:
-        match_type: regexp
+        # Use strict match type to ensure metrics like 'kustomize_resource_count'
+        # is excluded
+        match_type: strict
```

Comment:
The `kustomize.*` metrics are not included in the list. Do we need to change the match type?

Reply:
We don't as those are not meant to be exposed to users. The Kustomize metrics are shipped to Monarch only, not documented in our public doc.

Reply:
My question is whether we need to change the match_type from `regexp` to `strict`.

I thought `kustomize.*` metrics are not in the list anyway, so `regexp` would not include them.

Reply:
Oh I see, we were including `resource_count` from RG and matching with regex will include `kustomize_resource_count`.

Reply:
ACK